internal/gps/vcs_repo.go

```diff
@@ -31,6 +31,25 @@ type gitRepo struct {
 	*vcs.GitRepo
 }
 
+func newGitRepo(ustr, path string) (*gitRepo, error) {
+	r, err := vcs.NewGitRepo(ustr, path)
+	if err != nil {
+		if _, ok := err.(*vcs.LocalError); ok {
+			// if vcs could not initialize the repo due to a local error
+			// then the local repo is in an incorrect state. Remove and
+			// treat it as a new not-yet-cloned repo.
+			os.RemoveAll(path)
```

Comment:
@sectioneight i don't think there's a need to check for this particular error, since `os.RemoveAll` is the result of the parent error-handling.

If the folder could not be removed, then it might be better to return the parent error than this one.

Not a strong opinion though, let me know if there's a better reason to handle this error.